### PR TITLE
Combine two FE/server calls to get portal link into one

### PR DESCRIPTION
**Pull request by brendanm-stripe**
As discussed one on one

### CODE REVIEW

**Comment by brendanm-stripe** [comment]
PTAL @cjavilla-stripe  -- I added the other server languages here

TODO: fix the test

**Review by cjavilla-stripe — APPROVED** [review]
LGTM awesome work brendanm!

**Comment by brendanm-stripe** [comment]
Alright, turns out somebody was right about this whole "tests" thing. Latest commit updates the test pattern to work with the new sequence and includes server fixes.

Everything works with the rspec tests now except for non-slim PHP which doesn't seem to be compatible.

File: server/go/server.go [hunk 1]
```diff
@@ -115,34 +115,33 @@ func handleCustomerPortal(w http.ResponseWriter, r *http.Request) {
...
 		return
 	}
 
-	// This is the ID of the Stripe Customer and is typically stored in your
-	// database and would be retrieved along with the currently authenticated
-	// user. For demonstration, we're storing this value in the environment
-	// variables.
-	stripeCustomerID := req.CustomerID
+	// For demonstration purposes, we're using the Checkout session to retrieve the customer ID. 
+  // Typically this is stored alongside the authenticated user in your database.
```
Comment (brendanm-stripe) [inline server/go/server.go:127]: No idea why this comment line is indented weird. I don't see it misaligned locally 🤔

Comment (brendanm-stripe) [reply server/go/server.go:127]: Oh. Oh no. Tabs vs spaces. 😩

Comment (brendanm-stripe) [reply server/go/server.go]: Fixed!

Comment (cjavilla-stripe) [reply server/go/server.go]: in go, you can run `gofmt` and that'll format everything the same. Go has some special leading characters that aren't spaces. We usually use that everywhere except in the docs.

**Comment by cjavilla-stripe** [comment]
lgtm tho, feel free to merge when you're ready.

**Pull request merged**